ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/details/video/templates/options.html

```diff
@@ -0,0 +1,55 @@
+<!--/*
```

Comment:
I think this file might be better named? `options.html` is named as such for metadata properties since it contains all the "options" controls (select, checkbox, radio, etc.)